core/domain/story_fetchers_test.py

```diff
@@ -95,6 +95,21 @@ def test_get_story_by_id(self):
         story = story_fetchers.get_story_by_id(self.STORY_ID)
         self.assertEqual(story.to_dict(), expected_story)
 
+    def test_get_stories_by_ids(self):
```

Comment:
Is this test needed? Since, the test below tests this functionality as well.

Reply:
Yeah, the other one id ofr the None case. (I've changed the test name to make it distinct.)